lang/en/block_course_contents.php

```diff
@@ -46,4 +46,10 @@
 $string['config_enumerate_optional_on'] = 'Optional, enabled by default';
 $string['course_contents:addinstance'] = 'Add a new course contents block';
 $string['notusingsections'] = 'This course format does not use sections.';
-$string['pluginname'] = 'Course contents';
\ No newline at end of file
+$string['pluginname'] = 'Course contents';
+$string['config_enumerate_section_0'] = 'Enumerate section 0';
+$string['config_enumerate_section_0_desc'] = 'Enumerate the section 0 and number all sections appropriately.';
+$string['config_display_course_link'] = 'Display course page link';
+$string['config_display_course_link_desc'] = 'Display course home page link before all sections at the top.';
+$string['config_display_course_link_text'] = 'Display Course page link anchor text';
```

Comment:
Is there a use-case for not using the course name?

Reply:
Mainly because the course name might be too long.